### DISCUSSION TITLE
API to insert nested fields (many to many) [May 20]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2189] [Discord Threads]
> 🙋‍♂️ **Nils**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### Not at the moment. It will have to be done individually. However, this is something we will introrduce going forward  
#  
> 🙋‍♂️ **Nils**  
#### Is there a tracking ticket for this?  
#  
> 💁 **o1lab**  
#### just raised : https://github.com/nocodb/nocodb/issues/2034  
#  
> 💁 **o1lab**  
#### pls feel free to raise issue you feel are necessary  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/968862847197138954)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
